CONTRIBUTING.md

```diff
@@ -1,10 +1,22 @@
 # Contributing
 
+## Introduction
+
+This book is a catalogue of Rust programming techniques, (anti-)patterns, and idioms and other explanations.
+It is a compilation of collective (sometimes implicit) knowledge or experience that was created with collaborative effort.
```

Comment:
I'm not really sure how to phrase this, but maybe someone else has an idea?

Reply:
```suggestion
It is a compilation of collective (sometimes implicit) knowledge as well as experiences that have emerged through collaborative work.
```